ott/core/sinkhorn_lr.py

```diff
@@ -392,12 +390,10 @@ def norm_error(self) -> Tuple[int]:
 
   def _converged(self, state: LRSinkhornState, iteration: int) -> bool:
     criterion, count_escape, i, tol = state.criterion, state.count_escape, iteration, self.threshold
-    if i >= 2:
-      if criterion > tol / 1e-1:
+    if jnp.logical_not(i < 2):
```

Comment:
I don't think this will still work (locally tests fail for different reason); the way it's not can easily be rewritten purely using `jax.logical_{and,or,not}` without the Python `if`s.